Tests/Unit/ActiveDirectoryAuditRuleEntry.Tests.ps1

```diff
@@ -0,0 +1,592 @@
+#requires -Version 4.0 -Modules Pester
+
+#requires -RunAsAdministrator
+
+
+
+#region Setup for tests
+
+$Global:DSCModuleName = 'AccessControlDsc'
+
+$Global:DSCResourceName = 'ActiveDirectoryAuditRuleEntry'
+
+
+
+Import-Module "$($PSScriptRoot)\..\..\DSCResources\$($Global:DSCResourceName)\$($Global:DSCResourceName).psm1" -Force
+
+Import-Module "$($PSScriptRoot)\..\..\DscResources\AccessControlResourceHelper\AccessControlResourceHelper.psm1" -Force
+
+Import-Module Pester -Force
+
+#endregion
+
+#region Helper Functions
+
+function New-AuditAccessControlList
+{
+<#
+    .SYNOPSIS
```

Comment:
The other helper functions are placed in the testhelper module. If you guys like it better this way, we can add them to other test scripts my only opinion is that we stay consistent.

Reply:
I agree.  Additional functions should probably go in the helper